packages/docs/src/page-configs/ui-elements/data-table/examples/Sorting.vue

```diff
@@ -168,7 +168,7 @@ export default defineComponent({
     ]
 
     const columns = [
-      { key: 'id', sortable: true },
+      { key: 'id', sortable: true, sortingOptions: ['desc', 'asc'] },
```

Comment:
Don't you need to also add `null`?

Reply:
For example, we don't use `null` here as mentioned in issue #2119